### ISSUE TITLE
Tests?

### ISSUE THREAD

**Issue by jaredpalmer** [palmerhq/react-async-elements#12]

**Comment by echoghi** [comment]
Hey @jaredpalmer, I made a simple testing suite and would be interested in continuing if it looks alright. Testing the hooks seems pretty straightforward and we could wait on [Enzyme support](https://github.com/airbnb/enzyme/issues/1553) for `React.Suspense` to test the components, or use an alternate approach.

**Comment by DanielRuf** [comment]
> Hey @jaredpalmer, I made a simple testing suite and would be interested in continuing if it looks alright. Testing the hooks seems pretty straightforward and we could wait on [Enzyme support](https://github.com/airbnb/enzyme/issues/1553) for `React.Suspense` to test the components, or use an alternate approach.

This would be great. Is there an open PR?

**Comment by echoghi** [comment]
@DanielRuf Yup, I opened one here #28.

**Comment by echoghi** [comment]
@jaredpalmer Opened a new PR with a simple react-testing-library setup #43. I could add more tests, starting with the hooks if you're interested. Maybe look into Travis CI for automation as well re: @DanielRuf
